src/ProgramBlockEditor.js

```diff
@@ -484,7 +484,7 @@ class ProgramBlockEditor extends React.Component<ProgramBlockEditorProps, Progra
             if (this.state.replaceIsActive) {
                 this.props.focusTrapManager.setFocusTrap(
                     this.handleCloseReplaceFocusTrap,
-                    ['.replace-action-button', '.App__command-palette-command button'],
+                    ['.replace-action-button', '.focus-trap-replace-command'],
```

Comment:
Let's apply this naming consistently. Change ".replace-action-button" to ".focus-trap-replace-action-button". And also use this new naming convention on the other focus trap:

    ['.ProgramBlockEditor__program-block--pressed', '.ActionPanel__panel button'],